doc/developers/advanced_installation.rst

```diff
@@ -223,10 +223,27 @@ Finally you can build scikit-learn in the same ``cmd`` console::
 
     python setup.py install
 
-Replace ``/x64`` by ``/x86``  to build for 32-bit Python instead of 64-bit
+Replace ``/x64`` by ``/x86`` to build for 32-bit Python instead of 64-bit
 Python.
 
 
+Python >= 3.5
+-------------
+
+For Python versions as of 3.5, you need `Build Tools for Visual Studio 2017
+<https://visualstudio.microsoft.com/de/downloads/>`_.
+
+For 64-bit Python, configure the build environment with::
+
+    SET DISTUTILS_USE_SDK=1
+    "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
+
+And build scikit-learn from this environment::
+
+    python setup.py install
+
+Replace ``/x64`` by ``/x86`` to build for 32-bit Python.
```

Comment:
There is no `/` anymore, right? I think it is just: Replace ``x64`` by ``x86`` to build for 32-bit Python.